packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -0,0 +1,129 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import { CfnEnvironmentEC2 } from '../lib/cloud9.generated';
+
+/**
+ * A Cloud9 Environment
+ */
+export interface IEnvironmentEC2 extends cdk.IResource {
+  /**
+   * The name of the EnvironmentEc2
+   * @attribute
+   */
+  readonly environmentEc2Name: string;
+
+  /**
+   * The arn of the EnvironmentEc2
+   * @attribute
+   */
+  readonly environmentEc2Arn: string;
+
+}
+
+/**
+ * Properties for EnvironmentEc2
+ */
+export interface EnvironmentEC2Props {
+  /**
+   * The type of instance to connect to the environment.
+   *
+   * @default - t2.micro
+   */
+  readonly instanceType?: ec2.InstanceType;
+
+  /**
+   * The ID of the subnet in Amazon Virtual Private Cloud (Amazon VPC) that AWS Cloud9 will use to communicate with
+   * the Amazon Elastic Compute Cloud (Amazon EC2) instance. If you specify both `subnetId` and `vpc` properties,
+   * the `vpc` property will be ignored.
+   *
+   * @default - the first public subnet of the default VPC
+   */
+  readonly subnetId?: string;
```

Comment:
Why is this not a `SubnetSelection` object?

Reply:
fixed